src/librustc_save_analysis/dump_visitor.rs

```diff
@@ -1399,6 +1404,7 @@ impl<'l, 'tcx: 'l, 'll, O: DumpOutput + 'll> Visitor<'l> for DumpVisitor<'l, 'tc
                             span,
                             name: item.ident.to_string(),
                             value: String::new(),
+                            parent: None,
```

Comment:
I would have thought you'd want the module with the `extern crate` in would be the parent here, why is that not the case?